client/my-sites/stats/feedback/modal/index.tsx

```diff
@@ -19,6 +22,23 @@ interface ModalProps {
 
 const NOTICE_KEY_FOR_FEEDBACK_SUBMISSION = 'able_to_submit_user_feedback';
 
+const FEEDBACK_SHOULD_SHOW_PANEL_API_KEY = NOTICES_KEY_SHOW_FLOATING_USER_FEEDBACK_PANEL;
+const FEEDBACK_SHOULD_SHOW_PANEL_API_HIBERNATION_DELAY = 60; // 30 seconds for now
```

Comment:
TODO: Update this value to 12 months before merging and/or shipping.

Reply:
`30` in the comment might be typo?